default-templates/new-account/index.html

```diff
@@ -42,6 +42,10 @@ <h1>Data</h1>
         <span class="lead">Public Folder</span>
         <span class="badge">public</span>
       </a>
+      <a href="/public/" target="_blank" class="list-group-item">
```

Comment:
Cut'n'pastefail there?

Reply:
And this, this is why we do code reviews 😳